main.go

```diff
@@ -207,13 +208,22 @@ func run(state overseer.State) {
 	}
 
 	ctx := context.TODO()
-	e := engine.Start(ctx,
-		engine.WithConcurrency(*concurrency),
-		engine.WithDecoders(decoders.DefaultDecoders()...),
-		engine.WithDetectors(!*noVerification, engine.DefaultDetectors()...),
-		engine.WithDetectors(!*noVerification, conf.Detectors...),
-		engine.WithFilterUnverified(*filterUnverified),
-	)
+	var e *engine.Engine
+
+	if len(*detectors) > 0 {
+		e = engine.Start(ctx,
+			engine.WithConcurrency(*concurrency),
+			engine.WithDecoders(decoders.DefaultDecoders()...),
+			engine.WithDetectors(!*noVerification, engine.Detectors(ctx, strings.Split(*detectors, ","))...),
+		)
+	} else {
+		e = engine.Start(ctx,
+			engine.WithConcurrency(*concurrency),
+			engine.WithDecoders(decoders.DefaultDecoders()...),
+			engine.WithDetectors(!*noVerification, engine.DefaultDetectors()...),
+			engine.WithDetectors(!*noVerification, conf.Detectors...),
+		)
+	}
```

Comment:
**Suggestion:** I think the way this is setup wouldn't allow filters with custom detectors. Maybe the filter should be applied after the list of detectors have been configured?

Reply:
Ah okay. In my head i guess i was thinking if they had custom detectors they wouldn't want any of the non-custom detectors. I can potentially see wanting to use custom with select detectors too.